web/models/subject.ex

```diff
@@ -23,6 +23,57 @@ defmodule ActiveMonitoring.Subject do
     |> assoc_constraint(:campaign)
   end
 
+  def enroll_date(%Subject{inserted_at: inserted_at}) do
+    inserted_at
+  end
+
+  def first_call_date(%Subject{id: subject_id, campaign_id: campaign_id}) do
+    call = Repo.one(from c in Call,
```

Comment:
@mverzilli reworking these queries is a big issue. We're going to merge this as-is, and have filled #61 to not forget about it.

Reply:
We've added indices to help the queries we're currently executing, BTW.